hasher-matcher-actioner/hmalib/common/tests/test_action_rule_evaluation.py

```diff
@@ -51,3 +52,72 @@ def test_get_action_labels(self):
         action_labels = get_action_labels(match_message_with_foo, action_rules)
 
         assert len(action_labels) == 0
+
+        enqueue_mini_castle_for_review_action_label = ActionLabel(
+            "EnqueueMiniCastleForReview"
+        )
+        enqueue_sailboat_for_review_action_label = ActionLabel(
+            "EnqueueSailboatForReview"
+        )
+
+        action_rules = [
+            ActionRule(
```

Comment:
this rules are getting large enough that we might what to start defaulting to keyword args for clarity i.e.
``` 
ActionRule(
   action_label="Enqueue Mini-Castle for Review",
   must_have_labels=enqueue_mini_castle_for_review_action_label
   ...
)
```